packages/manager/modules/bm-server-components/src/network-tile/controller.js

```diff
@@ -11,6 +11,8 @@ export default class BmServerComponentsNetworkTileController {
 
   $onInit() {
     this.statePrefix = this.statePrefix || 'app.dedicated-server.server';
+    this.displayPublicBandwidth =
+      this.statePrefix !== 'nutanix.dashboard.nodes.node.general-info';
```

Comment:
You should rather define a property `hidePublicBandwitdth`  instead of checking the state name
```suggestion
```